documentation/modules/upgrading/proc-upgrade-cli-tool-direct.adoc

```diff
@@ -0,0 +1,110 @@
+// Module included in the following assemblies:
+//
+// assembly-upgrade-resources.adoc
+
+[id='proc-upgrade-cli-tool-direct-{context}']
+= Converting custom resources directly using the API conversion tool
+
+[role="_abstract"]
+This procedure describes how to use the API conversion tool to convert Strimzi custom resources directly in the kubernetes cluster into a format applicable to `{ApiVersion}`.
+To do so, you use the `convert-resource` (`cr`) command.
+The command uses Kubernetes APIs to make the conversions.
+
+You can specify one or more of types of Strimzi custom resources, based on the `kind` property, or you can convert all types.
+You can also target a specific namespace or all namespaces for conversion.
+When targeting a namespace, you can convert all custom resources in that namespace, or convert a single custom resource by specifying its name and kind.
+
+.Prerequisites
+
+* A Cluster Operator supporting the `{ApiVersion}` API version is up and running.
+* The API conversion tool, which is provided with the release artifacts.
+* The steps require a user admin account with RBAC permission to:
+** Get the Strimzi custom resources being converted using the `--name` option
+** List the Strimzi custom resources being converted without using the `--name` option
+** Replace the Strimzi custom resources being converted
+
+Use the CLI help for more information on the API conversion tool, and the flags available for the `convert-resource` command:
+
+[source,shell]
+----
+bin/api-conversion.sh help
+bin/api-conversion.sh help convert-resource
+----
+
+.Flags for converting custom resources
+[cols="2*",options="header",stripes="none"]
+|===
+
+|Flag
+|Description
+
+|`-k`, `--kind`
+|Specifies the kinds of custom resources to be converted, or converts all resources if not specified
+
+|`-a`, `--all-namespaces`
+|Converts custom resources in all namespaces
+
+|`-n`, `--namespace`
+|Specifies a Kubernetes namespace or OpenShift project, or uses the current namespace if not specified
+
+|`--name`
+|If `--namespace` and a single custom resource `--kind` is used, specifies the name of the custom resource being converted
+
+|`-ll`, `--log-level=<level>`
+|Sets the log level to enable logging
+
+|`-d`, `--debug`
+|Runs the tool in debug mode
+
+|===
+
+.Procedure
+
+. Run the API conversion tool with the appropriate command and flags.
++
+Example 1, converts all Strimzi resources in current namespace:
++
+[source,shell]
+----
+bin/api-conversion.sh convert-resource
```

Comment:
Same as in the previous procedure ... I wonder how much should we include the Windows variant. It probably makes no sense to have two versions for each command ... but maybe some mention might be worth it?

Reply:
I've added this after the _help_ instructions on each procedure: _Use `bin/api-conversion.cmd` on Windows._